Code commenter

daileyj93 opened the pull request:
Modified the script commenter by @CynicRus (PR#474) to be compatible with the "undo" command.
Comments or uncomments entire selection of code.
Ctrl + / to comment or uncomment slection - similar to Lazarus IDE.

CynicRus committed:
Add CynicRus's code commenter

ollydev committed:
Rework web. HTTPS fully working by default!

ollydev committed:
Use FPC resource files (Project Options > Resources) for OpenSSL binaries.

ollydev committed:
Fix debug build

ollydev committed:
Add TTarget.CopyData

daileyj93 committed:
Modified script commentor
Original from CynicRus/Simba:fpc-1.3
Now supports undo
Either comments or uncomments an entire code selection based on the first line of the selection

ollydev commented:
Does this only comment from the start of the line? It would be more in line with other IDEs (and Lazarus) if it comments vertically from the start X of the block.

smithtrenton commented on Projects/Simba/simbaunit.pas:
```diff
@@ -284,7 +285,8 @@   TSimbaForm = class(TForm)
     MTrayIcon: TTrayIcon;
     FunctionListHint: THintWindow;
-    procedure ActionColorsExecute(Sender: TObject);                                            
+    procedure ActCodeCommentExecute(Sender: TObject);
```
Why not keep naming consistent?

daileyj93 committed:
Revise code commenter
Comments start at first non-whitspace character of first line of selection, and form a straight column down.
Changed 'ActCodeComment' to 'ActionCodeComment' to remain consistent with naming convention for project

daileyj93 committed:
Merge branch 'simba-1.3' into code-commenter

MerlijnWajer commented:
Why was this closed?

daileyj93 commented:
I closed it because I screwed up the branch. I tried to rebase, but it totally corrupted the commit history. I guess you are not supposed to force push to an open PR. I re-opened this on PR#480. I hope this is okay.
